templates/az-oci-exa-pdb/README.md

```diff
@@ -142,5 +152,38 @@ terraform apply -var='region=us-ashburn-1' .....
+| vm_virtual_network_address_space                                 | The address space of the VM's virtual network. e.g. 10.3.0.0/16                                                                                                                                                                                                                                                      | `string` | Exa Infra and VM Cluster resource group |    no    |
+| vm_vnet_name                                                     | The virtual network name of the virtual machine.                                                                                                                                                                                                                                                                     | `string` | VM Cluster resource virtual network.    |    no    |
+| virtual_machine_name                                             | The name of the virtual machine.                                                                                                                                                                                                                                                                                     | `string` | n/a                                     |   yes    |
+| ssh_private_key_file                                             | The file path to SSH private key use to connect to VM.                                                                                                                                                                                                                                                               | `string` | n/a                                     |   yes    |
+| vm_public_ip_address                                             | Virtual machine public IP address.                                                                                                                                                                                                                                                                                   | `string` | ""      |    no    |
+| enable_connectivity_validation                                   | Enable or disable the CDB/PDB connectivity test.                                                                                                                                                                                                                                                                     | `bool`   | true    |    no    |
+
+# To Destroy VM after Connectivity Test
+
+Use `terraform destroy -target="module.virtual_machine"` to destroy the VM and its network resources.
```

Comment:
The limitation of "-target" option is it doesn't remove the module in the template files. Therefore, when the customer uses the same template files and create other resources ( e.g. 2nd VM Cluster ), the VM will still show in the terraform plan. But I don't think it's a blocking issue..

Reply:
Hi Ziqi, can you clarify ""-target" option is it doesn't remove the module in the template files" ? because during my test I can delete the vm via "-target" option and run terraform apply(with/without "-target" ) to create a new VM.

Reply:
I think my concern is on the template file itself. `-target` can remove the VM resource on Azure, but it doesn't edit the `main.tf`.  If the customer uses this command to destroy the VM only, it probably indicates they would like to reuse the same terraform state file and add other resources. And then it requires an extra step to remove the module in the `main.tf`.
In my opinion, validation could be in a separate template. But it's not worth the effort to do that change because of the reason mentioned above.